### PR TITLE
remove parking page for all tutorials

### PR DESCRIPTION
**Pull request by alexgeron-Luos**
⚠ PLEASE DO NOT DELETE THE TEXT BELOW ⚠

## Pull request's description

*Add here a description of the modifications you made.*

## Checklist before merging (please do not edit)
You must review you work before to submit the review to others.

### Self-review of your content
Remember the content must be readable and understandable by someone else than yourself.
- [ ] From a technical point of view.
- [ ] From a grammatical point of view (spelling mistakes, typos, clarity, etc.), and following the guidelines at the bottom.

### External reviews of your content
- [ ] You requested a technical review.
- [ ] You requested a grammatical review.
- [ ] You validated with @K0rdan or @alexgeron-Luos that it is safe to merge.

### Some guidelines to keep in mind
- Colons (:), semi-colons (;), exclamation (!), and interrogation points (?) are not preceded by a space (like full stops and commas). E.g.: Colons!
- File names and/or address are put in *italic*. E.g.: The file _main.c_.
- Functions, variables, or more generally short codes are put between grave accents. E.g.: To obtain `code()`, type \`code()\`.
- Long codes are put into blocks of code with three grave accent on each side, and the language's name:<br />
\`\`\`c<br />
// Some C language <br />
\`\`\`<br />

- "Luos engine" has a upper case on the **L** for "Luos" and lower case on the **e** for "engine".
- We call it "Luos engine" as a proper name, and ***not*** "the Luos engine".
- Following that fashion, anything that's owned by "Luos engine" implies that we must use `'s` as the standard English rule to show the possessive case, e.g. "Luos engine's API".
- The names pipe, gate, inspector, sniffer, app or application, driver, etc. have a lower case and can be refereed with the determiner `the`. E.g.: The inspector.

### CODE REVIEW

**Comment by lorenzi-nicolas** [comment]
- [ ] At the level of the card, remove the padding-top to raise the Tutorial, time and level logos and be at the same level as the title
- [ ] For each page, transform the link to match the original page (example for Your first project : /tutorials/get-started/get-started1 > /tutorials/get-started
- [ ] After that, make a 301 redirect between pages (example: /tutorials/get-started/get-started1 to /tutorials/get-started) to avoid a 404 (see table of redirections docs > luos.io/docs). Do this for all the pages concerned
- [ ] Switch trainings and time logos to white for dark mode
- [ ] Edit in the drop-down menu Espressif IDE > "Conect ESP to network" to "Connect ESP to network"

File: blog/2020-02-13-dc-motor-jungle.mdx [hunk 1]
```diff
@@ -37,4 +37,4 @@ If you liked what you read, please clap the hell out of it and <a rel="external
 
 Start learning how to control DC motors with this article: [How to stop being controlled by your DC motor: reverse the roles! Part 1](/2020-03-13-control-dc-motor-1.mdx)
 
-[Get Started with Luos](/tutorials/get-started)
+[Get Started with Luos](/tutorials/get-started))
```
Comment (K0rdan) [inline blog/2020-02-13-dc-motor-jungle.mdx:40]: Unnecessary `)`

File: blog/2022-08-12-how-to-develop-iot-edge-and-embedded-projects-easier.mdx [hunk 2]
```diff
@@ -66,7 +66,7 @@ Aerospace is a passion for many Luos developers and the cross-over was relevant.
 
 <h2>Focus on your main objectives and not on technical constraints</h2>
 
-The main objective is to get the rocket off the ground and exceed the altitude of 32km. The underlying objective of collecting the altitude and checking the flight plan was managed by Luos. The association teams increased their skills on Luos by performing the [Get started](https://www.luos.io/tutorials/get-started) and testing a proof of concept of a parachute release system.
+The main objective is to get the rocket off the ground and exceed the altitude of 32km. The underlying objective of collecting the altitude and checking the flight plan was managed by Luos. The association teams increased their skills on Luos by performing the [Get started](https://www.luos.io/tutorials/get-started and testing a proof of concept of a parachute release system.
```
Comment (K0rdan) [inline blog/2022-08-12-how-to-develop-iot-edge-and-embedded-projects-easier.mdx:69]: Missing `)` in the link

File: tutorials/arduino/arduino.mdx [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+hide_title: true
 image: /assets/images/Luos-with-Arduino-IDE-banner-luos.png
 description: Creating a Luos network with an Arduino board is very easy.
 ---
 
 import Image from '@site/src/components/Image';
 import IconExternalLink from '@theme/Icon/ExternalLink';
 
-# Use Luos with Arduino IDE
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/arduino/arduino.mdx:11]: Use the `@site` to reference components

File: tutorials/bike-alarm/bike-alarm.mdx [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+hide_title: true
+title: Develop a connected bike alarm
 image: /assets/images/Bike-alarm-Luos.png
+description: The purpose of the bike alarm is to create an alarm that continuously measures the movements of a bike.
 ---
 
-import Introduction from '@site/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/bike-alarm/bike-alarm.mdx:11]: Use `@site` to important components

File: tutorials/bike-alarm/cloud-alarm.mdx [hunk 5]
```diff
@@ -4,8 +4,9 @@ image: /assets/images/Bike-alarm-Luos.png
 ---
 
 import Image from '@site/src/components/Image';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/bike-alarm/cloud-alarm.mdx:7]: Use `@site` to import components

File: tutorials/bootloader/bootloader.mdx [hunk 6]
```diff
@@ -1,11 +1,23 @@
...
 custom_edit_url: null
+hide_title: true
+title: Bootloader
 image: /assets/images/Bootloader-Luos-banner.png
 ---
 
 import Image from '@site/src/components/Image';
 
-# Bootloader
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/bootloader/bootloader.mdx:10]: Use `@site` to import components

File: tutorials/esp/connect.mdx [hunk 7]
```diff
@@ -5,6 +5,7 @@ custom_edit_url: null
 import Image from '@site/src/components/Image';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/esp/connect.mdx:8]: Use `@site` to import components

File: tutorials/esp/connect.mdx [hunk 8]
```diff
@@ -124,7 +125,7 @@ You can now wire your ESP board to a Luos board containing a gate and a pipe ser
 Connect the network to the computer with a USB cable.
 
 :::info
-You can find more connection details in the tutorial [Get started: part 3](/tutorials/get-started/get-started3), where you connect the ESP board as a second board in a Luos network, with the pinout described in the section above.
+You can find more connection details in the tutorial [Get started: part 3](/tutorials/get-started)3), where you connect the ESP board as a second board in a Luos network, with the pinout described in the section above.
```
Comment (K0rdan) [inline tutorials/esp/connect.mdx:128]: Broken link

File: tutorials/esp/esp.mdx [hunk 9]
```diff
@@ -16,39 +17,95 @@ import Introduction from '/src/components/school/article/intro.js';
...
+# Part 1: Environment
+
+## 1. Introduction
+
+In this tutorial, we will learn how to perform the following essential actions:
+
+1. Configure the embedded environment on Espressif IDE.
+2. Flash a button service, create a Luos network and detect your service.
+
+<div align="center">
```
Comment (K0rdan) [inline tutorials/esp/esp.mdx:31]: Use Image component

File: tutorials/freertos/freertos.mdx [hunk 10]
```diff
@@ -1,11 +1,23 @@
...
 custom_edit_url: null
+title: 'How to use Luos engine with FreeRTOS'
+hide_title: true
 image: /assets/images/Luos-and-FreeRTOS-banner.png
 ---
 
 import Image from '@site/src/components/Image';
 
-# RTOS with Luos engine
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/freertos/freertos.mdx:10]: Use `@site` to import components

File: tutorials/get-started/get-started.mdx [hunk 11]
```diff
@@ -1,15 +1,16 @@
...
+description: This part will guide you through the basic essential tools' installation and how to use them.
 ---
 
-import Introduction from '/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+import IconExternalLink from '@theme/Icon/ExternalLink';
+import Form from '/src/components/typeform/index.js';
 
-#
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/get-started/get-started.mdx:13]: Use `@site` to import components

File: tutorials/get-started/get-started2.mdx [hunk 12]
```diff
@@ -145,7 +145,7 @@ See [Pyluos](/docs/integrations/pyluos) documentation for more information.
 
 ## 4. Next steps
 
-With your development environment installed in Part 1, you have now taken the control of a Luos app running on your MCU, with Python. The [next part](/tutorials/get-started/get-started3) of this section deals with creating your first Luos network.
+With your development environment installed in Part 1, you have now taken the control of a Luos app running on your MCU, with Python. The [next part](/tutorials/get-started)3) of this section deals with creating your first Luos network.
```
Comment (K0rdan) [inline tutorials/get-started/get-started2.mdx:148]: Broken link

File: tutorials/get-started/get-started3.mdx [hunk 13]
```diff
@@ -21,12 +21,12 @@ import TabItem from '@theme/TabItem';
 
 ## 1. Introduction
 
-The first two parts of the [Get started tutorial](/tutorials/get-started/) are now done. Good work! 💪
+The first two parts of the [Get started tutorial](/tutorials/get-started)) are now done. Good work! 💪
```
Comment (K0rdan) [inline tutorials/get-started/get-started3.mdx:24]: `)` not needed

File: tutorials/get-started/get-started3.mdx [hunk 14]
```diff
@@ -21,12 +21,12 @@ import TabItem from '@theme/TabItem';
...
 
 In this part, we will learn how to configure Luos engine to access to multiple boards in your physical network. Then, we will run our first example into these boards.
 
 :::info
-We will use the same board from the first two parts, but **you also need a second board to create a network**. Supported boards are listed [here](/tutorials/get-started/get-started1#setup-development-environment).
+We will use the same board from the first two parts, but **you also need a second board to create a network**. Supported boards are listed [here](/tutorials/get-started)#setup-development-environment).
```
Comment (K0rdan) [inline tutorials/get-started/get-started3.mdx:29]: Broken link

File: tutorials/get-started/get-started4.mdx [hunk 15]
```diff
@@ -7,12 +7,13 @@ description: In this part, we will learn how to add an inspector to your system,
 import Image from '@site/src/components/Image';
 import IconExternalLink from '@theme/Icon/ExternalLink';
 import Form from '/src/components/typeform/index.js';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/get-started/get-started4.mdx:10]: Use `@site` to import components

File: tutorials/get-started/get-started4.mdx [hunk 16]
```diff
@@ -7,12 +7,13 @@ description: In this part, we will learn how to add an inspector to your system,
...
 
 # Part 4: connect to a web app
 
 ## 1. Introduction
 
-In the previous parts of the [Get started tutorial](/tutorials/get-started/), we used Luos Python SDK (Pyluos) to be able to code the behavior of your embedded device from your computer. This SDK needs the gate application to translate JSON to Luos and Luos to JSON.
+In the previous parts of the [Get started tutorial](/tutorials/get-started)), we used Luos Python SDK (Pyluos) to be able to code the behavior of your embedded device from your computer. This SDK needs the gate application to translate JSON to Luos and Luos to JSON.
```
Comment (K0rdan) [inline tutorials/get-started/get-started4.mdx:16]: `)` not necessary

File: tutorials/luos-integration/install-ros2.mdx [hunk 17]
```diff
@@ -1,11 +1,22 @@
...
+title: How to use Luos engine with ROS
+hide_title: true
 custom_edit_url: null
 image: /assets/images/luos-integrations-install-ros-2-banner.png
 ---
 
 import IconExternalLink from '@theme/Icon/ExternalLink';
 
-# Install ROS 2 tutorial
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/luos-integration/install-ros2.mdx:10]: Use `@site` to import components

File: tutorials/luos-integration/ros1-retrocompatibility.mdx [hunk 18]
```diff
@@ -4,6 +4,7 @@ image: /assets/images/luos-integrations-ros1-retrocompatibility-banner.png
 ---
 
 import IconExternalLink from '@theme/Icon/ExternalLink';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/luos-integration/ros1-retrocompatibility.mdx:7]: Use `@site` to import components

File: tutorials/morse/add-service.mdx [hunk 19]
```diff
@@ -6,6 +6,7 @@ description: Now is the time to encode your own words to communicate in Morse co
 
 import Image from '@site/src/components/Image';
 import Form from '/src/components/typeform/index.js';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/morse/add-service.mdx:9]: Use `@site` to import components

File: tutorials/morse/morse.mdx [hunk 20]
```diff
@@ -18,29 +17,387 @@ import Introduction from '/src/components/school/article/intro.js';
...
+  <Image
+    src="/assets/images/tutorials/morse/morse-1-4.png"
+    darkSrc="/assets/images/tutorials/morse/morse-1-4-dark.png"
+  />
+</div>
+
+First, clone or download [this project](https://github.com/Luos-io/morse_encoder) and open it with VSCode:
+
+<div align="center">
+  <Image src="/assets/images/tutorials/morse/morse-1-5.png" darkSrc="" />
```
Comment (K0rdan) [inline tutorials/morse/morse.mdx:100]: Unnecessary `darkSrc` parameter

File: tutorials/pio/include.mdx [hunk 21]
```diff
@@ -6,6 +6,7 @@ description: The file _platformio.ini_ in your project describes the precompiler
 import Image from '@site/src/components/Image';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/pio/include.mdx:9]: Use `@site` to import components

File: tutorials/pio/pio.mdx [hunk 22]
```diff
@@ -1,15 +1,15 @@
...
+description: We will guide you through the process! You will learn how to set up a PlatformIO environment and how to make a good project structure.
 ---
 
-import Introduction from '/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-#
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/pio/pio.mdx:12]: Use `@site` to import components

File: tutorials/resilient-alias/resilient-alias.mdx [hunk 23]
```diff
@@ -1,10 +1,22 @@
 ---
 custom_edit_url: null
+title: How to have flexible and resilient aliases
+hide_title: true
 ---
 
 import Image from '@site/src/components/Image';
 
-# Resilient aliases management
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/resilient-alias/resilient-alias.mdx:9]: Use `@site` to import components

File: tutorials/your-first-detection/embedded-app.mdx [hunk 24]
```diff
@@ -8,6 +8,7 @@ import Image from '@site/src/components/Image';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Form from '/src/components/typeform/index.js';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/your-first-detection/embedded-app.mdx:11]: Use `@site` to import components

File: tutorials/your-first-detection/your-first-detection.mdx [hunk 25]
```diff
@@ -1,13 +1,16 @@
...
-description: In this tutorial, we will create an embedded application service using other services. We will move on to the next step, are you ready?
+description: In the previous training, we created a LED and a button services to control them using Pyluos.
 ---
 
-import Introduction from '@site/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/your-first-detection/your-first-detection.mdx:13]: Use `@site` to import components

File: tutorials/your-first-message/send-message.mdx [hunk 26]
```diff
@@ -8,6 +8,7 @@ import Image from '@site/src/components/Image';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Form from '/src/components/typeform/index.js';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/your-first-message/send-message.mdx:11]: Use `@site` to import components

File: tutorials/your-first-message/your-first-message.mdx [hunk 27]
```diff
@@ -1,15 +1,16 @@
...
+description: A button can be represented (as a LED) by a state-type service.
 ---
 
-import Introduction from '/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-#
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/your-first-message/your-first-message.mdx:13]: Use `@site` to import components

File: tutorials/your-first-service/create-a-package.mdx [hunk 28]
```diff
@@ -281,8 +282,9 @@ Then, use `pyluos-shell` to make a detection and you should see the following ro
 
 ## 5. Exercise
 
-Now try to move your led package on another board. Keep only the led package on this second board, and keep the pipe, the gate, and the blinker into the first one. Then wire a OneWire network as you learned in [Part 3 in the _Get started_](/tutorials/get-started/get-started3).
+Now try to move your led package on another board. Keep only the led package on this second board, and keep the pipe, the gate, and the blinker into the first one. Then wire a OneWire network as you learned in [Part 3 in the _Get started_](/tutorials/get-started)3).
```
Comment (K0rdan) [inline tutorials/your-first-service/create-a-package.mdx:285]: Broken link

File: tutorials/your-first-service/create-a-package.mdx [hunk 29]
```diff
@@ -8,6 +8,7 @@ import Image from '@site/src/components/Image';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Form from '/src/components/typeform/index.js';
+import Author from '/src/components/school/author/author.js';
```
Comment (K0rdan) [inline tutorials/your-first-service/create-a-package.mdx:11]: Use `@site` to import components

File: tutorials/your-first-service/your-first-service.mdx [hunk 30]
```diff
@@ -1,15 +1,16 @@
...
+description: In the Get started tutorial, we learned to perform the following essential actions
 ---
 
-import Introduction from '/src/components/school/article/intro.js';
+import Image from '@site/src/components/Image';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-#
+import Introduction from '/src/components/school/article/intro.js';
```
Comment (K0rdan) [inline tutorials/your-first-service/your-first-service.mdx:13]: Use `@site` to import components

File: tutorials/your-first-service/your-first-service.mdx [hunk 31]
```diff
@@ -18,59 +19,357 @@ import Introduction from '/src/components/school/article/intro.js';
...
+1. Set up the embedded environment to make a basic Luos-LED-blinking project.
+2. Control of your microservices with high-level API and set up the Python environment.
+3. How to spread microservices across multiple boards on a network while keeping the exact same behavior with Luos engine.
+
+Upon finishing the _Get started_, you can grab what a microservice represents into an embedded system. However, Luos still remains black magic to you.
+
+Let us demystify it by helping you to make first service code!
+
+<div align="center">
+  <img
```
Comment (K0rdan) [inline tutorials/your-first-service/your-first-service.mdx:39]: Use `Image` component

File: tutorials/your-first-service/your-first-service.mdx [hunk 32]
```diff
@@ -18,59 +19,357 @@ import Introduction from '/src/components/school/article/intro.js';
...
+    ┃  │  Type                Alias               ID   ┃
+    ┃  ├> State               led                 2    ┃
+    ┃  ├> Pipe                Pipe                3    ┃
+    ┃  ├> Gate                gate                1    ┃
+    ┃  ╰> Unknown             blinker             4    ┃
+  ╔>┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+```
+
+<div align="center">
+  <img
```
Comment (K0rdan) [inline tutorials/your-first-service/your-first-service.mdx:183]: Use `Image` component

**Review by K0rdan — CHANGES_REQUESTED** [review]

**Review by K0rdan — APPROVED** [review]

**Pull request merged**